go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/creack/pty v1.1.18
 	github.com/denisbrodbeck/machineid v1.0.1
 	github.com/enriquebris/goconcurrentqueue v0.6.3
+	github.com/gomarkdown/markdown v0.0.0-20221013030248-663e2500819c // indirect
```

Comment:
You need to run `go mod tidy` once, and commit the changes

Reply:
done

Reply:
makes no difference, there's no tagged release for this lib.